mkdocs/tests/config/config_options_tests.py

```diff
@@ -80,6 +80,24 @@ def test_length(self):
                           option.validate, "Testing Long")
 
 
+class ChoiceTest(unittest.TestCase):
+
+    def test_valid_choice(self):
+        option = config_options.Choice(('yes', 'no'))
```

Comment:
`yes` and `no` seem like good general strings to use for testing. However, we may never see them in actual use as the YAML parser will convert them to bool values (I think). This doesn't really require a change to the test but it might be less confusing to someone reading the source who doesn't know about YAMl's type handling if we use something else.

Not a necessary change, but might be nice to have.